### PR TITLE
Error to explaining warning when loading lpmd messages in checker

### PR DESCRIPTION
**Pull request by JanHoefelmeyer**
solves https://github.com/csaf-poc/csaf_distribution/issues/415

### CODE REVIEW

File: cmd/csaf_checker/processor.go [hunk 1]
```diff
@@ -1217,7 +1217,9 @@ func (p *processor) checkProviderMetadata(domain string) bool {
 
 	for i := range lpmd.Messages {
 		// TODO: Filter depending on the role.
```
Comment (s-l-teichmann) [inline cmd/csaf_checker/processor.go:1219]: The TODO should be removed.

**Review by s-l-teichmann — CHANGES_REQUESTED** [review]

File: cmd/csaf_checker/processor.go (hunk 1 above)
Comment (JanHoefelmeyer) [reply cmd/csaf_checker/processor.go]: Done via https://github.com/csaf-poc/csaf_distribution/pull/429/commits/16041bdb897aa9208f2c438cb0a521e7d3337b53

**Review by bernhardreiter — APPROVED** [review]
LGTM

**Review by s-l-teichmann — APPROVED** [review]
LGTM & works

**Pull request merged**